### PR TITLE
doc: add RafaelGSS to Security Team (patches/reports)

### PR DESCRIPTION
**Pull request by RafaelGSS**
cc: @mhdawson

### CODE REVIEW

**Review by mcollina — APPROVED** [review]
lgtm

**Review by Trott — CHANGES_REQUESTED** [review]
Line 56 is fine, but the other list should be updated with `ncu-team sync Security-Team.md` instead.

**Review by Trott — APPROVED** [review]

**Comment by Trott** [comment]
> Line 56 is fine, but the other list should be updated with `ncu-team sync Security-Team.md` instead.

I opened https://github.com/nodejs/TSC/pull/1217 to deal with that, so landing this first is fine.

File: Security-Team.md [hunk 1]
```diff
@@ -88,6 +89,7 @@ the Node.js program on HackerOne.
 * [@mmarchini](https://github.com/mmarchini) - mary marchini
 * [@MylesBorins](https://github.com/MylesBorins) - Myles Borins
 * [@RaisinTen](https://github.com/RaisinTen) - Darshan Sen
+* [@RafaelGSS](https://github.com/RafaelGSS) - Rafael Gonzaga
```
Comment (Trott) [inline Security-Team.md:92]: Whoops, this is out of order and should have gone before `@RaisinTen`. But that's OK, it should be fixed by #1217 and is a trivial issue regardless.

**Pull request merged**